modules/administration_manual/pages/installation/manual_installation_ubuntu_16_04.adoc

```diff
@@ -0,0 +1,163 @@
+....
+After this has finish, create the database and a database user for ownCloud to use:
+
+....
+mysql -u root
+
+MariaDB [(none)]> create database owncloud;
+MariaDB [(none)]> GRANT USAGE ON owncloud.* TO owncloud@localhost IDENTIFIED BY 'password';
+MariaDB [(none)]> GRANT ALL PRIVILEGES ON owncloud.* TO owncloud@localhost IDENTIFIED BY 'password';
+MariaDB [(none)]> flush privileges;
+MariaDB [(none)]> exit
+....
+
+=== Install Redis Server
+Redis in needed for Transactional File Locking in ownCloud. Perform the following step to install it:
+....
+apt install -y redis-server
+....
+
+=== Install Apache and the necessary dependencies
+The following steps will install the Apache Webserver and the dependencies needed for ownCloud:
+....
+apt install -y \
+apache2 \
+libapache2-mod-php \
+php-gd \
+php-json \
+php-mysql \
+php-curl \
+php-intl \
+php-mcrypt \
+php-imagick \
+php-zip \
+php-xml \
+php-mbstring \
+php-soap \
+php-ldap \
+php-apcu \
+php-redis \
+smbclient \
+php-smbclient \
+gettext-base && \
+a2enmod ssl rewrite headers env dir mime ldap && \
+service apache2 restart
+....
+
+=== Configure Apache
+We will create a new apache config file for ownCloud:
+....
+vim /etc/apache2/sites-available/owncloud.conf
+....
+Please add the following to the file and adjust parameters like SERVER-NAME and
+the paths to the certificates.
+....
+<VirtualHost *:80>
+    ServerAdmin webmaster@localhost
+    ServerName  SERVER_NAME
+    Redirect permanent / https://SERVER_NAME/
+</VirtualHost>
+
+<IfModule mod_ssl.c>
+    <VirtualHost _default_:443>
+        ServerAdmin webmaster@localhost
+        ServerName  SERVER_NAME
+        DocumentRoot /var/www/owncloud/
+        <Directory />
+                Options FollowSymLinks
+                AllowOverride All
+        </Directory>
+
+        <IfModule mod_headers.c>
+                Header always set Strict-Transport-Security "max-age=15768000;  includeSubDomains; preload"
+        </IfModule>
+        ErrorLog ${APACHE_LOG_DIR}/error.log
+
+        # Possible values include: debug, info, notice, warn, error, crit,
+        # alert, emerg.
+        LogLevel warn
+
+        SSLEngine on
+        SSLProtocol all -SSLv2 -SSLv3
+        SSLCipherSuite EDH+CAMELLIA:EDH+aRSA:EECDH+aRSA+AESGCM:EECDH+aRSA+SHA25 6:EECDH:+CAMELLIA128:+AES128:+SSLv3:!aNULL:!eNULL:!LOW:!3DES:!MD5:!EXP:  !PSK:!DSS:!RC4:!SEED:!IDEA:!ECDSA:kEDH:CAMELLIA128-SHA:AES128-SHA
+        SSLHonorCipherOrder on
+        SSLCompression off
+        SSLCertificateFile      /path/to/certificate
+        SSLCertificateKeyFile   /path/to/key
+        CustomLog /var/log/apache2/ssl_request_log "%t %h %{SSL_PROTOCOL}x  %{SSL_CIPHER}x \"%r\" %b"
+
+        BrowserMatch "MSIE [2-6]" nokeepalive ssl-unclean-shutdown  downgrade-1.0 force-response-1.0
+        BrowserMatch "MSIE [17-9]" ssl-unclean-shutdown
+
+    </VirtualHost>
+</IfModule>
+....
+
+After that enable the new config and reload Apache:
+....
+a2ensite owncloud
```

Comment:
there are several other modules we have to enable:

a2enmod headers
a2enmod env
a2enmod dir
a2enmod mime
a2enmod unique_id